doc/iterations/2023.8/shapes/customize_containers_with_border.adoc

```diff
@@ -0,0 +1,55 @@
+= (M) Add border style for containers
+
+== Problem
+
+In forms, it is difficult to identify the limits of a container (group or flexbox).
+
+== Key Result
+
+Give a specifier the possibility to add to each container a style with border.
+
+== Solution
+
+In the View DSL, add to containers eClassifiers a reference to `borderStyle`.
+Interpret this style to render the container with the correct border representation on the UI.
+Like the other existing styles, the specifier can add multiple conditional border style.
+
+=== Scenario
+
+When a specifier adds a container, he can define a style to choose the representation of his border.
+On the end user UI, the container has the chosen representation for the container's border.
+
+=== Breadboarding
+
+Border style properties :
+
+image::images/customize_containers_with_border_01.png[border_style_properties]
+
+Conditional border style properties :
+
+image::images/customize_containers_with_border_02.png[conditional_border_style_properties]
+
+UI representation with some default border :
+
+image::images/customize_containers_with_border_03.png[form_with_borders]
+
+=== Cutting backs
+
+Do not offer the possibility of declaring conditional styles on these containers.
+
+== Rabbit holes
+
+* There are two types of container in the View DSL :
+** Group
+** Flexbox
+* Reuse of existing `BorderStyle` in the view DSL, this style declares 4 attributs :
+** borderColor
+** borderRadius
+** borderSize
+** borderLineStyle
+* For a container, the `borderStyle` is optional and doesn't exist by default.
+* There can be only one `borderStyle`, but several conditional styles.
+* The conditional border style condition property is an expression that will be computed in the context of the container's semantic element.
```

Comment:
Not sure why these points are "rabbit holes".

Reply:
I would simply add that by reusing `BorderStyle` which has been created for the diagram DSL, you are exposing the view DSL to some minor risks. If / when we will want to change things for one of those DSLs, we will have to change it for everybody (for example, when we will allow specifier to indicate different size, color, style of border for the top, left, bottom, right borders).

I'm not against this but we should just have it in mind, especially when we will split the view DSL in X parts.

Reply:
I moved most of these points to the solution part.